<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagerForm</class>
 <widget class="QDialog" name="PackagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>PackagerForm</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>131</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="newBttn">
      <property name="text">
       <string>New package</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadBttn">
      <property name="text">
       <string>Load package...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveBttn">
      <property name="text">
       <string>Save package...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addBttn">
      <property name="text">
       <string>Add Files...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="expBttn">
      <property name="text">
       <string>Export Files...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="remBttn">
      <property name="text">
       <string>Remove Files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chngBttn">
      <property name="text">
       <string>Change Key...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="prevBttn">
      <property name="text">
       <string>Preview...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeBttn">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
